dpnp/backend/include/dpnp_iface.hpp

```diff
@@ -457,6 +457,22 @@ INP_DLLEXPORT void dpnp_prod_c(void* result_out,
                                const void* initial,
                                const long* where);
 
+/**
+ * @ingroup BACKEND_API
+ * @brief Range of values (maximum - minimum) along an axis.
+ *
+ * @param [out] result      Output array.
+ * @param [in]  array       Input array with data.
+ * @param [in]  res_shape   Shape of input array.
+ * @param [in]  res_ndim    Number of elements in shape.
+ * @param [in]  shape       Shape of input array.
+ * @param [in]  ndim        Number of elements in shape.
+ * @param [in]  axis        Axis.
+ * @param [in]  naxis       Number of elements in axis.
+ */
+template <typename _DataType>
+void dpnp_ptp_c(void* result1, void* array1_in, const size_t* res_shape, size_t res_ndim, const size_t* shape, size_t ndim, const size_t* axis, size_t naxis);
```

Comment:
Let's make interface of the function similar to interface of `dpnp_matmul_c`. I mean result and input parameters. Parameters `axis` look ok.